create_graphics.py

```diff
@@ -332,19 +336,33 @@ def parse_args():
         )
 
     # Ensemble panel-specific args
-    map_group = parser.add_argument_group('Ensemble Panel Arguments')
-    map_group.add_argument(
+    ens_group = parser.add_argument_group('Ensemble Panel Arguments')
+    ens_group.add_argument(
```

Comment:
This was mostly unrelated, and I have no idea what kinds of issues it may have caused!